optional_plugins/runner_remote/avocado_runner_remote/__init__.py

```diff
@@ -571,36 +571,36 @@ def configure(self, parser):
             return
 
         msg = 'test execution on a remote machine'
-        self.remote_parser = run_subcommand_parser.add_argument_group(msg)
-        self.remote_parser.add_argument('--remote-hostname',
-                                        dest='remote_hostname', default=None,
-                                        help='Specify the hostname to login on'
-                                        ' remote machine')
-        self.remote_parser.add_argument('--remote-port', dest='remote_port',
-                                        default=22, type=int, help='Specify '
-                                        'the port number to login on remote '
-                                        'machine. Current: 22')
-        self.remote_parser.add_argument('--remote-username',
-                                        dest='remote_username',
-                                        default=getpass.getuser(),
-                                        help='Specify the username to login on'
-                                        ' remote machine. Current: '
-                                        '%(default)s')
-        self.remote_parser.add_argument('--remote-password',
-                                        dest='remote_password', default=None,
-                                        help='Specify the password to login on'
-                                        ' remote machine')
-        self.remote_parser.add_argument('--remote-key-file',
-                                        dest='remote_key_file', default=None,
-                                        help='Specify an identity file with '
-                                        'a private key instead of a password '
-                                        '(Example: .pem files from Amazon EC2)')
-        self.remote_parser.add_argument('--remote-timeout', metavar='SECONDS',
-                                        help=("Amount of time (in seconds) to "
-                                              "wait for a successful connection"
-                                              " to the remote machine. Defaults"
-                                              " to %(default)s seconds."),
-                                        default=60, type=int)
+        remote_parser = run_subcommand_parser.add_argument_group(msg)
+        remote_parser.add_argument('--remote-hostname',
+                                   dest='remote_hostname', default=None,
+                                   help=('Specify the hostname to login on'
+                                         ' remote machine'))
```

Comment:
Well you don't have to use brackets when splitting lines while already inside a bracket, which allows longer lines and is easier to read. Anyway it was mixed before, now you unify it (the other way than I'd prefer) so let's just proceed, but IMO it looks uglier...